invenio_records_files/__init__.py

```diff
@@ -172,9 +177,10 @@
 
 Integration with invenio-records-ui
 -----------------------------------
-If you are using ``invenio-records-ui``, you can easily add new views by
-defining new endpoints into your ``RECORDS_UI_ENDPOINTS`` configuration.
-In particular, you can add the ``file_download_ui`` endpoint:
+If you are using `Invenio-Records-UI <https://invenio-records-ui.RTFD.io/>`_,
+you can easily add new views by defining new endpoints into your
+``RECORDS_UI_ENDPOINTS`` configuration. In particular, you can add the `
+`file_download_ui`` endpoint:
```

Comment:
`file_download_ui` is not rendering well as code, I think the backticks need to be together (there is a line break between the opening ones).